nifty-client/pom.xml

```diff
@@ -72,6 +72,13 @@
         </dependency>
 
         <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <version>1.1.33.Fork17</version>
```

Comment:
Don't put versions in submodules -- all versions should go in the root POM's `dependencyManagement` section.